spyder/app/utils.py

```diff
@@ -143,8 +140,8 @@ def filter(self, record):
             root_logger.addHandler(handler)
 
 
-def delete_lsp_log_files():
-    """Delete previous dead Spyder instances LSP log files."""
+def delete_debug_log_files():
```

Comment:
This could be moved to `plugins/application/utils.py`, since it's not necessary in the main window anymore.